ticker_http_handler.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+)
+
+type tickerHTTPHandler struct {
+	ts *TickerService
+	mw *MeasurementsWriter
+}
+
+func newTickerHTTPHandler(ds *DeviceService) tickerHTTPHandler {
+	measurements := make(chan Measurement, 10)
+
+	ts := TickerService{ds: ds, measurements: measurements}
+	ds.AddObserver(&ts)
+
+	mw := MeasurementsWriter{measurements: measurements}
+
+	return tickerHTTPHandler{ts: &ts, mw: &mw}
+}
+
+func (h *tickerHTTPHandler) Start(w http.ResponseWriter, _ *http.Request) {
+	if err := h.ts.Start(); err != nil {
+		log.Print(err)
+		http.Error(w, "failed to start ticker", http.StatusInternalServerError)
+	}
+
+	h.mw.Start()
+
+	w.WriteHeader(http.StatusOK)
```

Comment:
just a small thing - in case of error you first wrire ok header and then you set it for the second time when there was and error. It should not matter, I agree:) but for some reason (hidden in http server:) golang compains about it.